eventhubs/client/azure-eventhubs/src/main/java/com/azure/eventhubs/EventHubClientBuilder.java

```diff
@@ -3,13 +3,19 @@
 
 package com.azure.eventhubs;
 
+import com.azure.core.amqp.TransportType;
+import com.azure.core.exception.AzureException;
 import com.azure.core.http.policy.RetryPolicy;
+import com.azure.eventhubs.implementation.ReactorProvider;
+import com.azure.eventhubs.implementation.SharedAccessSignatureTokenProvider;
 import reactor.core.scheduler.Scheduler;
 
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
 import java.time.Duration;
 
 /**
- * Builder to create an {@link EventHubClient}.
+ * Builder to createReactor an {@link EventHubClient}.
```

Comment:
Not sure if this was intended;  it may be a language idiom that I'm not familiar with or it may have been caught up unintentionally in a refactoring.

Reply:
Fixed. IntelliJ tries to refactor and replace values in docs as well as code. In this case, it didn't realise there wasn't any relation.